lib/perfectNumber.js

```diff
@@ -10,7 +10,20 @@
  * @returns {boolean}
  */
 function perfectNumber(num) {
-  return true;
+  var sum = 0;
+  var upperLimit = Math.floor(num / 2);
+
+  // There are no odd perfect numbers.
+  if (num % 2 != 0) {
+    return false;
+  }
```

Comment:
Thanks for the contribution. I'm glad you were able to figure this out. This predicate has already been made as `isOdd`. It's a quick `require` away! Reusing previous work is an important part of developing on a team and building a library.

Reply:
Thank you for the advice. I have made the requested changes.